soql-analyzer/src/main/scala/com/socrata/soql/AnalysisSerializer.scala

```diff
@@ -289,6 +290,7 @@ class AnalysisSerializer[C,T](serializeColumn: C => String, serializeType: T =>
       writeGrouped(isGrouped)
       writeDistinct(analysis.distinct)
       writeSelection(selection)
+      maybeWrite(from) { x => out.writeStringNoTag(x) }
```

Comment:
Joins aren't really in active use.  Therefore we can avoid the dance.  We all ignored version difference when we added outer joins.